storage_scrubber/README.md

```diff
@@ -1,4 +1,4 @@
-# Neon S3 scrubber
+# Neon Storage Scrubber
 
 This tool directly accesses the S3 buckets used by the Neon `pageserver`
```

Comment:
This still mentions S3, but it's correct since the scrubber doesn't support anything else atm :shrug:

Reply:
Yeah it's not the intended end state.